README.md

```diff
@@ -147,16 +155,19 @@ See file `Makefile` which currently supports cross compilation. Just run `make`
 
 Also inspect file `bin/md5Sums.txt` after the cross compilation.
 
-
-## Configuration
+### API keys
 
 Optionally set the `GOOGLE_API_CLIENT_ID` and `GOOGLE_API_CLIENT_SECRET` environment variables to use your own API keys.
 
+### Scripts
+
+Installing scripts which offer automated drive commands including syncing is described in [README-Automation.md](https://github.com/odeke-em/drive/blob/master/README-Automation.md).
```

Comment:
Let's hold up until https://github.com/odeke-em/drive/pull/873#issuecomment-278129493 is addressed and merged, as we'll also need to change the name.
Aside from that PR being a dependency, this change is 99% ready to fly.